include/odp/api/spec/packet_io_stats.h

```diff
@@ -26,42 +26,23 @@ extern "C" {
  */
 
 /**
- * Packet IO statistics
- *
- * Packet IO statistics counters follow RFCs for Management Information Base
- * (MIB)for use with network management protocols in the Internet community:
- * https://tools.ietf.org/html/rfc3635
- * https://tools.ietf.org/html/rfc2863
- * https://tools.ietf.org/html/rfc2819
+ * Packet IO statistics counters
  */
 typedef struct odp_pktio_stats_t {
-	/**
-	 * The number of octets in valid MAC frames received on this interface,
-	 * including the MAC header and FCS. See ifHCInOctets counter
-	 * description in RFC 3635 for details.
-	 */
+	/** Number of octets in successfully received packets (includes MAC
+	 *  header and FCS). */
 	uint64_t in_octets;
```

Comment:
I think the notion of "successfully" needs to be clarified. E.g. if statistics is gathered on the MAC/hardware level, then successfully received (by hardware) packets may include good and bad packets, those which will be sent to application and dropped. So, successfully may be specified as "non-dropped good and bad", or "non-dropped good (no errors) only", "all, including dropped,  good or bad", etc.

Reply:
Updated in V4